dags/openshift_nightlies/util/manifest.py

```diff
@@ -46,8 +46,24 @@ def get_baremetal_releases(self):
                     "versionAlias": version_alias,
                     "profile": profile
                 })
+            
+    def get_openstack_releases(self):
+        for version in self.yaml['platforms']['openstack']: 
```

Comment:
change this to `for version in self.yaml['platforms'].get('openstack', [])` so that manifests with no defined releases for openstack don't error out.

Reply:
fixed